README.md

```diff
@@ -55,7 +55,7 @@ const { GenericContainer } = require("testcontainers");
 (async () => {
   const buildContext = path.resolve(__dirname, "my-dir");
   
-  const container = await GenericContainer.fromDockerfile(buildContext);
+  const container = await GenericContainer.fromDockerfile(buildContext).build();
```

Comment:
Will need a major release for this as it's a breaking change

Reply:
Unfortunately yes. This was the cause I previously chose a different style, but I don't see another way :P

Reply:
I also don't see any other way 👍 though I do think the builder pattern is the best way to go